firstmoversadvantage/fma-ruby-on-rails-template
# 174213960 update rails and gems versions

#### What's this PR do?
- Upgrades gems & packages versions and fixes several minor errors in html syntax

#### How should this be manually tested?
- Setup application from scratch and make sure it all works (see README)
- Click through application, make sure all links and forms work (see db/seeds for user credentials)
- Run tests

#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/174213960

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
